spec/schemas/ml._common.yaml

```diff
@@ -59,12 +84,74 @@ components:
         _id:
           type: string
           description: The hit ID.
+        _version:
```

Comment:
I believe this is shared from a base object, so it should become an `allOf:` with a base response that contains `_seq_no` and `_primary_term` and then the fields that are specific to this object here.